docs/tutorial/coreutils.rst

```diff
@@ -146,16 +146,16 @@ Available encryption algorithms
 Available compression algorithms
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo:: Provide relative throughput numbers compare to none.
+.. todo:: Provide relative throughput numbers compared to »none«.
 
 +----------------+----------------------------------------------------------------------------+
 | NAME           |   DESCRIPTION                                                              |
 +================+============================================================================+
-| ``snappy``     | High throughput, relative low compression rate.                            |
+| ``snappy``     | High throughput, relative low compression ratio.                           |
 +----------------+----------------------------------------------------------------------------+
-| ``lz4``        | High throughput, slightly higher compression rate than snappy.             |
+| ``lz4``        | High throughput, slightly higher compression ratio than snappy.            |
 +----------------+----------------------------------------------------------------------------+
-| ``guess``      | Choose best algorithm based on file ending, size and mime type.            |
+| ``guess``      | Chooses best algorithm based on file ending, size and mime type.           |
```

Comment:
I am thought about it a bit more. The "*the* best" implies the absolute best. While here we are actually guessing based on prior knowledge.

How about:
"Picks a compression algorithm based on file extension, size, and mime type."

By the way do we really use `size`? It is used as a threshold to do compression or not, but I think it is uniform for all algorithm. I.e. it would be enough to say somewhere above "Files smaller than ... are not compressed"

Reply:
> How about:
> "Picks a compression algorithm based on file extension, size, and mime type."

I replaced "best" with "suitable".

> By the way do we really use size? It is used as a threshold to do compression or not, but I think it is uniform for all algorithm. I.e. it would be enough to say somewhere above "Files smaller than ... are not compressed"

Sure, but we might use the size at some point. Some compression formats have higher overhead for each block (zstd vs zlib e.g.). There it makes sense to choose another algorithm until a certain size. For the user it's not that important what attributes we actually use for the heuristic and how exactly the guessing works. From my experience to not go into detail too much in documentation. That also saves us from updating (or forgetting to do so) whenever we change the heuristic.

Reply:
I like your reasoning. Let's go with it.